fgcm/fgcmGray.py

```diff
@@ -919,8 +919,9 @@ def makeExpGrayPlots(self):
         expNGoodStars = snmm.getArray(self.expNGoodStarsHandle)
         expGray = snmm.getArray(self.expGrayHandle)
 
-        expUse,=np.where((self.fgcmPars.expFlag == 0) &
-                         (expNGoodStars > self.minStarPerExp))
+        expUse, = np.where((self.fgcmPars.expFlag == 0) &
```

Comment:
There are comments in the `np.where()` documentation suggesting the use of `np.nonzero()`, but I don't think it applies in this case where the condition is complex.